src/App.tsx

```diff
@@ -1,10 +1,118 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { USER_ID, getTodos, addTodo, deleteTodo } from './api/todos';
+
+import { Todo } from './types/Todo';
+import { TodoList } from './сomponents/TodoList';
+import { Footer } from './сomponents/Footer';
+
+export enum Filter {
+  ALL = 'All',
+  ACT = 'Active',
+  COMP = 'Completed',
```

Comment:
Move enum for the separate file
```suggestion
export enum Filter {
  All = 'All',
  Active = 'Active',
  Completed = 'Completed',
```